Internal/MQTT-Client-Framework/MQTTClientGJ/MQTTClientGJ/MQTTLog.h

```diff
@@ -3,11 +3,13 @@
 // Copyright © 2013-2017, Christoph Krey. All rights reserved.
 //
 
-#if __has_feature(modules)
-    @import Foundation;
-#else
-    #import <Foundation/Foundation.h>
-#endif
+//#if __has_feature(modules)
```

Comment:
check why this code is commented out

Reply:
<img width="1309" alt="Screenshot 2023-09-20 at 9 23 09 AM" src="https://github.com/gojek/courier-iOS/assets/66404008/1ec5ccae-e016-4f63-9141-aec8d30fed9f">

This code was commented out because Xcode throws error when `@ import` is used in Frameworks , as mentioned in this guide - https://developer.apple.com/documentation/xcode/identifying-and-addressing-framework-module-issues , 

Will delete this code and push a commit.